packages/button-theme/stories/Link.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react/types-6-0';
+import Link from '../src/Link';
+import { HtmlOnlyWrapper } from '../../../stories/helpers';
+
+export default {
+  title: 'Link',
+  component: Link,
+  argTypes: {
+    size: {
+      control: {
+        type: 'select',
+        options: ['small', 'medium', 'large'],
+      },
+      description: 'The size of the link',
+    },
+    href: {
+      description: 'The url that the hyperlink points to.',
+    },
+    external: {
+      description: 'Link to an external page.',
+    },
+  },
+} as Meta;
+
+const Template: Story = args => (
+  <Link {...args} href={args.href}>
```

Comment:
If I don't explicitly put in `href`, even though it's part of the `...args`, I get an error. Thoughts?